config/default/assisted-service-patch-volumes.yaml

```diff
@@ -10,7 +10,8 @@ spec:
         app: assisted-service
     spec:
       containers:
-      - env:
-        - name: DEPLOY_TARGET
-          value: k8s
-        name: assisted-service
\ No newline at end of file
+        volumeMounts:
```

Comment:
Also update the patch name and comment in https://github.com/openshift/assisted-service/blob/master/config/default/kustomization.yaml#L65.